lib/utils/application_util.ex

```diff
@@ -25,4 +25,15 @@ defmodule EaRestaurantDataLoader.Lib.Utils.ApplicationUtil do
   def decode_json(json_data) do
     Poison.decode(json_data)
   end
+
+  def get_string_bson_object_id(bson_object_id) do
+    BSON.ObjectId.encode(bson_object_id)
+  end
+
+  def update_document_id_to_string(document) do
+    %{_id: bson_object_id} = document
+    {:ok, string_object_id} = get_string_bson_object_id(bson_object_id)
```

Comment:
I think you don't need to create a function for this. Also I think it is better to create a `mongo_db_util.ex` file for this purpoe

Reply:
the Library to parse to json dont know how to read BSON objecto and parse it to string, for this reason a convert the `_id` to string before to parse to json.

Reply:
That sounds good, but that's why we have a layer for mappers, because you can do any kind of transformations there